config/spark-defaults.conf

```diff
@@ -31,3 +31,5 @@ spark.jars                      /home/spark-current/jars/azure-storage-2.0.0.jar
 # Note: Default filesystem master HA
 spark.deploy.recoveryMode       FILESYSTEM
 spark.deploy.recoveryDirectory  /root/
+
+spark.shuffle.service.enabled   true
```

Comment:
as part of this change, should we also move the default directory off of the OS drive? It feels a bit worrying to me that we're intentionally going to put data there.